2-collectors/scala-stream-collector/project/metals.sbt

```diff
@@ -0,0 +1,4 @@
+// DO NOT EDIT! This file is auto-generated.
+// This file enables sbt-bloop to create bloop config files.
+
+addSbtPlugin("ch.epfl.scala" % "sbt-bloop" % "1.4.0-RC1-105-118a551b")
```

Comment:
do we need that?

Reply:
Sorry! Added to global git ignore, so it won't happen again!